### PR TITLE
Bug 1434595 Added registry storage configuration to the advanced install topic

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1434595

cc @sferich888 

Will verify in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Verified with tech review and QA. @openshift/team-documentation PTAL

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -722,6 +722,92 @@ openshift_examples_modify_imagestreams=true
...
+#openshift_hosted_registry_storage_s3_bucket=bucket_name
+#openshift_hosted_registry_storage_s3_region=bucket_region
+#openshift_hosted_registry_storage_s3_chunksize=26214400
+#openshift_hosted_registry_storage_s3_rootdirectory=/registry
+#openshift_hosted_registry_pullthrough=true
+#openshift_hosted_registry_acceptschema2=true
+#openshift_hosted_registry_enforcequota=true
+----
+
+If using a different S3 service, such as Minio or ExoScale, also add the region endpoint parameter:
```
Comment (gaurav-nelson) [inline install_config/install/advanced_install.adoc]: s/If using/If you are using

**Comment by gaurav-nelson** [comment]
Please also check xrefs, looks like some are broken.

**Comment by gaurav-nelson** [comment]
@bfallonf LGTM. few comments though. :fish:

**Comment by bfallonf** [comment]
Thanks @gaurav-nelson . Changed to suggested. I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Installing a Cluster -> Advanced Installation]
|Added the xref:../install_config/install/advanced_install.adoc#advanced-install-registry-storage[Configuring Registry Storage] section.
%

**Pull request merged**